src/Controls/tests/DeviceTests/Elements/Modal/ModalTests.Windows.cs

```diff
@@ -27,13 +27,22 @@ await CreateHandlerAndAddToWindow<IWindowHandler>(new Window(navPage),
 				async (handler) =>
 				{
 					var windowRootViewContainer = (WPanel)handler.PlatformView.Content;
+
+#pragma warning disable RS0030 // Do not use banned APIs; Panel.Children is banned for performance reasons. In tests, we don't mind.
+					var windowRootViewContainerChildren = windowRootViewContainer.Children;
+#pragma warning restore RS0030 // Do not use banned APIs
```

Comment:
Wait, do the device tests use `BannedApiAnalyzers`?!? Seems like tests should not.

Reply:
This is what controls it:
https://github.com/dotnet/maui/blob/ed53c15169c75679dcb079a841e67f6212fa6b59/Directory.Build.targets#L17

Seems like `$(IsTestProject)` should be `true` for the DeviceTests. Is it?